include/seqan3/alignment/configuration/align_config_parallel.hpp

```diff
@@ -7,14 +7,13 @@
 
 /*!\file
  * \brief Provides seqan3::align_cfg::parallel configuration.
- * \author Lydia Buntrock <lydia.buntrock AT fu-berlin.de>
- * \author Jörg Winkler <j.winkler AT fu-berlin.de>
+ * \author Svenja Mehringer <svenja.mehringer AT fu-berlin.de>
```

Comment:
you shouldn't change the authorship; you can add youself to that

Reply:
Alright. I just thought I literally changed every line of code so they wouldn't recognise their code and should not be held responsible for it. :innocent: